tests/appsec/sca/test_heartbeat.py

```diff
@@ -0,0 +1,13 @@
+import string
+from utils import BaseTestCase, interfaces
+from utils.tools import logger
+
+
+class Test_SCA_heartbeat(BaseTestCase):
+    def test_status_ok(self):
+        """Test a heartbeat is sent"""
+
+        interfaces.library.expected_timeout = 90
```

Comment:
The heartbeat from telemetry will be tested in dedicated telemetry test (and we'll use `DD_TELEMETRY_HEARTBEAT_INTERVAL` to avoid any trick with `expected_timeout`) so you can safely remove this test.